Backend/somethingpt2.html

```diff
@@ -19,10 +19,8 @@
 </form>
 <style>
     form {
-        /* Center the form on the page */

```

Comment:
Some comments we can leave; while not all of them apply, some comments can be helpful to understand what certain styles are doing